docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-SOAPCALL.xml

```diff
@@ -446,8 +446,16 @@
 END;
 ip := 'http://127.0.0.1:8022/';
 ips := 'https://127.0.0.1:8022/';
-ipspw := 'https://username:password@127.0.0.1:8022/';
 svc := 'MyModule.SomeService';
+ips_secret := 'secret:myConnectSecret';
+        /* assumes a secret names myConnectSecret exists and contains:
```

Comment:
"named".  The secret name in kubernetes or the vault would actually be "http-connect-myConnectSecret".